pkg/handler/trigger.go

```diff
@@ -360,6 +367,15 @@ func (h *PublicHandler) triggerAsyncNamespaceModel(ctx context.Context, req Trig
 		return nil, err
 	}
 
+	userUID := uuid.FromStringOrNil(resource.GetRequestSingleHeader(ctx, constant.HeaderUserUIDKey))
+	if err = h.modelUsageHandler.Check(ctx, &usage.ModelUsageHandlerParams{
+		UserUID:      userUID,
+		OwnerUID:     ns.NsUID,
+		RequesterUID: uuid.FromStringOrNil(resource.GetRequestSingleHeader(ctx, constant.HeaderRequesterUID)),
+	}); err != nil {
+		return nil, status.Error(codes.FailedPrecondition, err.Error())
```

Comment:
Same comment as in [L115](https://github.com/instill-ai/model-backend/pull/619/files#diff-344ee1e543402cfc79c30cb124c14b2b2aa9c52cf1768b9f9eb71ac97fbb8d50R115)

Reply:
fixed